### ISSUE TITLE
Create env fails with `Platform.script` pointing to wrong directory

### ISSUE THREAD

**Issue by Anas35** [pingbird/puro#28]
After installing with Power shell, I test it out with
```
$ puro create my_env stable --log-level=4
 [D] PURO_FLUTTER_BIN: null
[D] Config: PuroConfig(
      gitExecutable: LocalFile: 'git',
      puroRoot: LocalDirectory: 'C:\Users\moham\.puro',
      pubCacheDir: LocalDirectory: 'C:\Users\moham\.puro\shared\pub_cache',
      homeDir: LocalDirectory: 'C:\Users\moham',
      projectDir: null,
      parentProjectDir: null,
      flutterGitUrl: https://github.com/flutter/flutter.git,
      engineGitUrl: https://github.com/flutter/engine.git,
      releasesJsonUrl: https://storage.googleapis.com/flutter_infra_release/releases/releases_windows.json,
      flutterStorageBaseUrl: https://storage.googleapis.com,
      environmentOverride: null,
      puroBuildsUrl: https://puro.dev/builds,
      buildTarget: PuroBuildTarget.windowsX64,
      enableShims: false,
    )
[D] target: PuroBuildTarget.windowsX64
[D] Platform.executable: puro
[D] Platform.resolvedExecutable: C:\Users\moham\.puro\bin\puro.exe
[D] Platform.script: file:///C:/Users/moham/puro
[D] Platform.packageConfig: null
[W] Error while resolving Platform.script
    FileSystemException: Cannot resolve symbolic links, path = 'C:\Users\moham\puro' (OS Error: The system cannot find the file specified.
    , errno = 2)
    #0      FileSystemEntity._throwIfError (dart:io/file_system_entity.dart:839)
    #1      FileSystemEntity.resolveSymbolicLinksSync (dart:io/file_system_entity.dart:358)
    #2      ForwardingFileSystemEntity.resolveSymbolicLinksSync (package:file/src/forwarding/forwarding_file_system_entity.dart:56)
    #3      PuroVersion.provider.<anonymous closure> (package:puro/src/version.dart:113)
    #4      LazyProviderNode.value (package:puro/src/provider.dart:58)
    #5      _LinkedHashMapMixin.[]= (dart:collection-patch/compact_hash.dart)
    #6      RootScope.read (package:puro/src/provider.dart:75)
    #7      PuroVersion.of (package:puro/src/version.dart:223)
    #8      _installTrampoline (package:puro/src/install/bin.dart:96)
    #9      ensurePuroInstalled (package:puro/src/install/bin.dart:44)
    #10     EnvCreateCommand.run (package:puro/src/commands/env_create.dart:42)
    #11     CommandRunner.runCommand (package:args/command_runner.dart:212)
    #12     PuroCommandRunner.runCommand (package:puro/src/command.dart:342)
    #13     CommandRunner.run.<anonymous closure> (package:args/command_runner.dart:122)
    #14     new Future.sync (dart:async/future.dart:302)
    #15     CommandRunner.run (package:args/command_runner.dart:122)
    #16     main (package:puro/src/cli.dart:234)
    #17     _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295)
    #18     _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:192)

[D] No package root: Platform.packageConfig is null
[D] pubInstallBinDir: c:\users\moham
[D] pubInstallPackageDir: c:\users
[D] packageRoot: null
[D] executablePath: c:\users\moham\.puro\bin\puro.exe
[D] scriptPath: c:\users\moham\puro
[D] scriptExtension:
[D] installationType: PuroInstallationType.unknown
[D] version: 1.3.3
[V] #0      _installTrampoline (package:puro/src/install/bin.dart:127)
    <asynchronous suspension>
    #1      ensurePuroInstalled (package:puro/src/install/bin.dart:44)
    <asynchronous suspension>
    #2      EnvCreateCommand.run (package:puro/src/commands/env_create.dart:42)
    <asynchronous suspension>
    #3      CommandRunner.runCommand (package:args/command_runner.dart:212)
    <asynchronous suspension>
    #4      main (package:puro/src/cli.dart:234)
    <asynchronous suspension>

[x] Can't install puro: Could not determine installation method
```

Did I somehow messed up the installtion?

**Comment by pingbird** [comment]
Thanks for the bug report! I was able to reproduce this with cmd.exe, there is a bug in Dart that causes Platform.executable to point to the wrong place: https://github.com/dart-lang/sdk/issues/45005

This should be fixed in 1.3.4

**Comment by Anas35** [comment]
Thanks for the quick fix. Tested out on 1.34 and its working.
